modules/snooker/src/main/ts/ephox/snooker/queries/Direction.ts

```diff
@@ -0,0 +1,15 @@
+import { Fun } from '@ephox/katamari';
+import { Direction } from '@ephox/sugar';
+
+const ltr = {
+  isRtl: Fun.constant(false)
```

Comment:
`Fun.never` instead please and likewise `Fun.always` below instead of `Fun.constant(bool)`.

Reply:
I have no idea why I changed this... Thanks!

Reply:
You probably just copied the old version before I fixed it about 2 weeks ago :shrug:

Reply:
Probably XD